connector/src/main/java/org/geysermc/connector/skin/SkinProvider.java

```diff
@@ -412,14 +412,22 @@ private static Skin supplyEars(Skin existingSkin, String earsUrl) {
 
         // if the requested image is a cape
         if (provider != null) {
-            if (image.getWidth() > 64) {
+            if (image.getWidth() > 64 || image.getHeight() > 32) {
                 // Prevent weirdly-scaled capes from being cut off
                 BufferedImage newImage = new BufferedImage(128, 64, BufferedImage.TYPE_INT_ARGB);
                 Graphics g = newImage.createGraphics();
                 g.drawImage(image, 0, 0, image.getWidth(), image.getHeight(), null);
                 g.dispose();
                 image.flush();
                 image = scale(newImage, 64, 32);
+            } else if (image.getWidth() < 64 || image.getHeight() < 32) {
```

Comment:
```suggestion
            } else {
```

Reply:
The reason I did not carry over this change is because we don't need to scale capes that are already 64 x 32.

Reply:
My bad I didn't see that it checked both height and width.